metricbeat/Makefile

```diff
@@ -22,21 +22,22 @@ include ${ES_BEATS}/libbeat/scripts/Makefile
 # Collects all module dashboards
 .PHONY: kibana
 kibana:
-	-rm -r etc/kibana
+	# To not remove index-pattern as generated by update
+	-rm -r etc/kibana/dashboard etc/kibana/search etc/kibana/visualization
 	mkdir -p etc/kibana
 	-cp -r module/*/_meta/kibana etc/
 
 # Collects all module and metricset fields
 .PHONY: fields
 fields:
 	mkdir -p etc/
-	cat ${ES_BEATS}/metricbeat/etc/_meta/fields_base.yml > etc/fields.yml
-	. ${PYTHON_ENV}/bin/activate; python ${ES_BEATS}/metricbeat/scripts/fields_collector.py >> etc/fields.yml
+	cat ${ES_BEATS}/metricbeat/etc/fields.yml > etc/fields.generated.yml
```

Comment:
I like `fields.common.yml` for the Beats where it only contains a subset of the fields. I like this name because it makes it clear that it's not the complete fields.yml.